bin/pg_tapgen

```diff
@@ -107,11 +124,30 @@ sub get_schemas {
 
 sub schemas_are {
     my $schemas = shift;
+    return unless @$schemas;
+    my @schemas = @$schemas;
+    unless ('public' ~~ @schemas ) {
```

Comment:
I don't understand the need for this. The `get_schemas()` function will return public unless it was included in `--exclude-schema`. With this block it gets included even if it's in `--exclude-schema`, which I don't think is right.